x/launch/keeper/msg_server_settle_request_test.go

```diff
@@ -16,6 +16,10 @@ func TestMsgSettleRequest(t *testing.T) {
 		addr3                       = sample.Address()
 		addr4                       = sample.Address()
 		addr5                       = sample.Address()
+		addr6                       = sample.Address()
```

Comment:
Can we generate the addresses here in a array and the `[]RequestSample` array below in a `for` loop?